source/jormungandr/tests/instant_system_new_default_routing_tests.py

```diff
@@ -205,3 +206,61 @@ def test_basic_ride_sharing(self):
 
         assert rsj_sections[2].get('type') == 'crow_fly'
         assert rsj_sections[2].get('mode') == 'walking'
+
+    def test_ride_sharing_with_pt(self):
+        """
+        test ridesharing_jouneys details
+        """
+        q = journey_basic_query + \
+            "&last_section_mode[]=walking" + \
+            "&first_section_mode[]=ridesharing" + \
+            "&ridesharing_speed=2.5" + \
+            "&debug=true"
```

Comment:
you should not use this debug and still have the result, we have to remove it for tests.
But you will have to provide a mock response for the ridesharing to prevent it being filtered